notion-core/src/test/scala/com/ambiata/notion/core/LocationIOSpec.scala

```diff
@@ -120,6 +121,17 @@ class LocationIOSpec extends Specification with ScalaCheck { def is = s2"""
     e <- i.exists(p)
   } yield e ==== true)
 
+  def copy = prop((from: LocationTemporary, to: LocationTemporary, lines: List10[N]) =>
+    for {
+      f  <- from.location
+      i  <- from.io
+      _  <- i.writeUtf8Lines(f, lines.value.map(_.value))
```

Comment:
Could you either change this to be a blob of raw bytes or add another test. This limits the test a bit more than it should I think.